src/Lib/Managers/ItemsManager.vala

```diff
@@ -51,33 +49,35 @@ public class Akira.Lib.Managers.ItemsManager : Object {
         window.event_bus.change_item_z_index.connect (on_change_item_z_index);
     }
 
-    public Models.CanvasItem? insert_item (Gdk.EventButton event) {
+    public Models.CanvasItem? insert_item (double x, double y) {
```

Comment:
This change makes the loading of files way easier, great job!

Reply:
Yes, I did in order to make the testing phase easier (Idk how to create an EventButton). 

Good to know it'll help in other contexts as well!